lib/renderers.js

```diff
@@ -19,13 +19,26 @@ function mustache(engine, str, locals, options) {
 }
 
 function twig(engine, str, locals, options) {
-  var opt = merge({ data: str }, options)
-  var template = engine.twig(opt)
+  const opt = merge({ data: str }, options)
+
+  // This part add all twig files dependencies
+  // to make webpack able to watch them
+  const registry = [];
+  engine.extend((Twig) => {
+    const defaultSave = Object.assign(Twig.Templates.save);
+    Twig.Templates.save = function customSave(template) {
+      registry.push(template.path);
+      return defaultSave.call(this, template);
+    };
+  });
+  const template = engine.twig(opt)
+  registry.forEach(this.addDependency);
+
```

Comment:
Is there a straightforward way to [write a test](https://github.com/jabney/render-template-loader/blob/master/test/twig.spec.js) for this functionality?

Reply:
I currently don't know how to write this test but I agréé, de should add it.